nautobot_ssot_infoblox/utils/nautobot.py

```diff
@@ -26,6 +26,10 @@ def get_prefix_vlans(prefix: Prefix) -> list:
     Returns:
         list: List of VLAN objects with RelationshipAssociation to passed Prefix.
     """
+    vlan_list = []
     pf_relations = prefix.get_relationships()
     pf_vlan_relationship = Relationship.objects.get(name="Prefix -> VLAN")
-    return [x.destination for x in pf_relations["source"][pf_vlan_relationship]]
+    if pf_vlan_relationship in pf_relations["source"]:
```

Comment:
could "source" generate a `KeyError`?

Reply:
I don't believe so since all Relationships have the source and destination keys. Going off the code in core [here](https://github.com/nautobot/nautobot/blob/ae9024b2f3b3667d2f432a5e1ca7c3257812f908/nautobot/extras/models/relationships.py#L63) appears to confirm that.